models/aged_creditor.json

```diff
@@ -7,21 +7,21 @@
             "attributes": {
                 "code": {
                     "column": "demarcation_code",
-                    "label": "Code",
+                    "label": "Demarcation Code",
                     "type": "string",
                     "description": "Demarcation Code desc"
```

Comment:
While you're at it, why not make this:

Municipality code assigned by the Municipal Demarcation Board